Sources/CocoaLumberjack/DDLog.m

```diff
@@ -985,6 +989,8 @@ - (instancetype)initWithMessage:(NSString *)message
                             tag:(id)tag
                         options:(DDLogMessageOptions)options
                       timestamp:(NSDate *)timestamp {
+    NSParameterAssert(message && file);
```

Comment:
Please use two `NSParameterAssert`s for the two parameters cause the description for a failed assert mentions the parameter name.

Reply:
Committed.